bigquery_etl/metadata/parse_metadata.py

```diff
@@ -405,8 +411,14 @@ class DatasetMetadata:
     dataset_base_acl: str = attr.ib()
     user_facing: bool = attr.ib(False)
     labels: Dict = attr.ib({})
+    default_table_workgroup_access: Optional[List[Dict[str, Any]]] = attr.ib(None)
     workgroup_access: list = attr.ib(DEFAULT_WORKGROUP_ACCESS)
 
+    def __attrs_post_init__(self):
+        """Set default table workgroup access to workgroup access."""
```

Comment:
Actually, it appears that this block makes it so that a default value is always available, which means the logic in most recent suggestion won't do anything. I suppose that explains some earlier logic that was writing this file all the time redundantly. It seems fine to do that but there are probably ways to avoid it as well.